### ISSUE TITLE
Combine, and AsyncSequence support

### ISSUE THREAD

**Issue by Prince2k3** [merindorium/Guava#12]
Will you this library every support this?

**Comment by Terky** [comment]
Could you provide an example of what are you wanted to see?

**Comment by Prince2k3** [comment]
Definitely! I have a version working with Combine. I'll make a PR to get more feedback.

**Comment by Prince2k3** [comment]
This is a non issue closing....
